src/plugins/telemetry/server/telemetry_collection/get_local_stats.test.ts

```diff
@@ -14,6 +14,7 @@ import {
   createCollectorFetchContextMock,
```

Comment:
NIT: Changes to this file are not actually needed, but I had to go through these because I was having weird `bootstrap` errors. It turned out it was due to the local bootstrap cache. Happy to revert the changes if anyone thinks they should.